lineage/BaumWelch.py

```diff
@@ -144,6 +146,31 @@ def do_M_E_step(tHMMobj, gammas):
     for state_j in range(tHMMobj.num_states):
         tHMMobj.estimate.E[state_j].estimator(all_cells, all_gammas[:, state_j])
 
+def do_M_E_step_1state(all_tHMMobj, gammas_1st, state_j):
+    """ perform the M_E step when all the concentrations are given at once for one state.
+    all_cells is a list (for different concentrations) containing lists of tuples of observations. 
+    gammas_1st is a list of lists, each cell's gamma value for one state. """
+    G1cells = []
+    G2cells = []
+    for tHMMobj in all_tHMMobj:
+        all_cells = np.array([cell.obs for lineage in tHMMobj.X for cell in lineage.output_lineage])
+        G1cells.append(all_cells[:, np.array([0, 2, 4])])
```

Comment:
This is collecting the emissions, right? Given that these don't depend on the state, why not do this in `do_M_E_step_atonce` rather than `do_M_E_step_1state`?

Reply:
I removed `do_M_E_step_1state` and all is included in the `do_M_E_step_atonce` now.